Lib/asyncio/streams.py

```diff
@@ -655,9 +655,9 @@ async def read(self, n=-1):
 
         If n is zero, return empty bytes object immediately.
 
-        If n is positive, this function try to read `n` bytes, and may return
+        If n is positive, this function tries to read `n` bytes, and may return
         less or equal bytes than requested, but at least one byte. If EOF was
-        received before any byte is read, this function returns empty byte
+        received before any byte is read, this function returns an empty bytes
```

Comment:
I suggest porting the refined version from the above here as well.

Reply:
I will do this once we've agreed on the refined version.

Reply:
Looks like this was done in [python/cpython@`a920fbe` (#101807)](https://github.com/python/cpython/pull/101807/commits/a920fbed365b97152e638d906eb237d7414464c1)